vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -1023,11 +1047,53 @@ void WindowsTextInputComponentView::DrawText() noexcept {
         static_cast<LONG>(offset.x) + static_cast<LONG>(m_imgWidth),
         static_cast<LONG>(offset.y) + static_cast<LONG>(m_imgHeight)};
 
-    winrt::check_hresult(m_textServices->OnTxInPlaceActivate(&rcClient));
-
-    // TODO keep track of proper invalid rect
-    auto hrDraw = m_textServices->TxDrawD2D(d2dDeviceContext.get(), &rc, nullptr, TXTVIEW_ACTIVE);
-    winrt::check_hresult(hrDraw);
+    // draw text ontop of richedit
+    if (!m_placeholderText.empty() && m_firstTextUpdate <= 2) {
+      m_firstTextUpdate++; // TODO: Figure out why DrawText is/needs to be called twice to actually draw
+      winrt::com_ptr<ID2D1SolidColorBrush> brush;
+      winrt::check_hresult(
+          d2dDeviceContext->CreateSolidColorBrush(D2D1::ColorF(D2D1::ColorF::Green, 1.0f), brush.put()));
+
+      facebook::react::LayoutConstraints constraints;
+      constraints.maximumSize.width = static_cast<FLOAT>(offset.x) + static_cast<FLOAT>(m_imgWidth);
+      constraints.maximumSize.height = static_cast<FLOAT>(offset.y) + static_cast<FLOAT>(m_imgHeight);
+
+      // Create a fragment with text attributes
```

Comment:
Ideally you'd create and cache these only when anything that would affect them changes -- not everytime they are drawn.